bin/collator/src/parachain/service.rs

```diff
@@ -397,14 +409,15 @@ where
     let is_authority = parachain_config.role.is_authority();
     let prometheus_registry = parachain_config.prometheus_registry().cloned();
     let transaction_pool = params.transaction_pool.clone();
-    let import_queue = cumulus_client_service::SharedImportQueue::new(params.import_queue);
+    // let import_queue = cumulus_client_service::SharedImportQueue::new(params.import_queue);
```

Comment:
Looks like this import queue shared with something, in other case let's cleanup comment.

Reply:
Good catch